src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -95,7 +95,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
     navigation.navigate('TopicPageScreen', navigationParam);
   };
 
-  const openChat = (screen: string) => {
+  const openChat = (screen: string, from?: string) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `openChat` function now accepts an optional parameter `from`. This is a good change as it allows the function to be more flexible and adaptable to different use cases. However, it's important to ensure that all calls to this function are updated accordingly, especially if they rely on the previous function signature.
```diff
-  const openChat = (screen: string) => {
+  const openChat = (screen: string, from?: string) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->